src/layouts/BasicLayout.js

```diff
@@ -98,6 +98,16 @@ class BasicLayout extends React.PureComponent {
     }
     return title;
   }
+  getBashRedirect = () => {
+    // According to the url parameter to redirect
+    // 这里是重定向的,重定向到 url 的 redirect 参数所示地址
+    const urlParams = new URL(window.location.href);
```

Comment:
check if you can use this.props.location instead - using window.  makes code harder to test.